mysql-test/suite/rocksdb/t/drop_table-master.opt

```diff
@@ -1,4 +1,3 @@
 --rocksdb_max_subcompactions=1
---rocksdb_info_log_level=info_level
```

Comment:
@alxyang Do you have an issue to track this? I have a similar issue in Percona Server that I have not yet reported to Facebook as I have not ruled out some problem as part of our port. Our tracking issue is here https://jira.percona.com/browse/MYR-127